_data/trans/gr.json

```
@@ -0,0 +1,41 @@
+{	
+	"banner": "Δοκιμάστε το νέο μας <span>Chrome Extension</span>",
```

```suggestion
    "banner": "Δοκιμάστε το νέο μας <span>Chrome Extension</span>",
```
Use 4 spaces instead of tabs. This applies to all indented lines.